docker/build_scripts/finalize.sh

```diff
@@ -28,8 +28,12 @@ for PREFIX in $(find /opt/_internal/ -mindepth 1 -maxdepth 1 -name 'cpython*');
 	ln -s ${PREFIX} /opt/python/${ABI_TAG}
 	# Make versioned python commands available directly in environment.
 	PYVERS=$(${PREFIX}/bin/python -c "import sys; print('.'.join(map(str, sys.version_info[:2])))")
-	ln -s ${PREFIX}/bin/python /usr/local/bin/python${PYVERS}
-	ln -s ${PREFIX}/bin/pip /usr/local/bin/pip${PYVERS}
+	if [[ "${PREFIX}" == *"/pypy"* ]]; then
+		ln -s ${PREFIX}/bin/python /usr/local/bin/pypy${PYVERS}
+	else
+		ln -s ${PREFIX}/bin/python /usr/local/bin/python${PYVERS}
+		ln -s ${PREFIX}/bin/pip /usr/local/bin/pip${PYVERS}
```

Comment:
I now wonder if the pip link should be removed or not ?

Introduced in #1090 when adding python symlinks.

Reply:
If `pip` is symlinked for cpython, it should be symlinked for pypy too. 

I personally prefer to explicitly use `python -m pip`, but many people seem to prefer to trust that `pip` is properly symlinked.

Reply:
> If pip is symlinked for cpython, it should be symlinked for pypy too.

But we can't symlink `pip3.7` both for cpython 3.7 and pypy 3.7 in `/usr/local/bin/`

> I personally prefer to explicitly use `python -m pip`, but many people seem to prefer to trust that `pip` is properly symlinked.

I do too.
For the sake of consistency and given the fact that pipx.y were introduced very recently, I'm going to remove them.

Reply:
Thinking back on your comment, I think you might have had `ln -s pip3.7 /opt/python/.../pip` or `ln -s pip3.7 /opt/python/.../pip` in mind rather than adding a link to PyPy's pip in `/usr/local/bin` ?

If that's the case then maybe your comment will change ?

Mine still applies in any case (unless someone objects to removing those pip symlinks in `/usr/local/bin`

Reply:
Yes, this was not talking about the pipX.Y, but I'm not attached to those either, pythonX.Y -m pip or pypy3.7 -m pip is better. :)

Reply:
`pipX.Y` symlinks in `/usr/local/bin` have been removed in master.
`pythonX.Y -m pip` or `pypyX.Y -m pip` can be used.